oracle/blockoracle.go

```diff
@@ -153,12 +153,12 @@ func (bo *MinerBlockOracle) getLatestATXID() (types.AtxId, error) {
 	atxIDs, err := bo.activationDb.GetNodeAtxIds(bo.nodeID)
 	if err != nil {
 		bo.log.Info("did not find ATX IDs for node: %v, error: %v", bo.nodeID.Key[:5], err)
-		return types.AtxId{common.HexToHash("11111111")}, err
+		return types.AtxId{}, err
```

Comment:
this weirs string implanted as atx id helps us track theorigin of the null id for now... consider leaving it

Reply:
I thought this was left behind unintentionally after debugging. You think leaving this is a good idea?